Makefile.Common

```diff
@@ -3,10 +3,19 @@
 # otherwise in the example command pipe, only the exit code of `tee` is recorded instead of `go test` which can cause
 # test to pass in CI when they should not.
 SHELL = /bin/bash
-ifeq ($(shell uname -s),Windows)
```

Comment:
This check was added by #17183. However, I couldn't find a *nix shell for Windows that will return "Windows". Moreover, none of the popular `bash.exe` for Windows accepts '/' as a switch prefix, they treat it as a path. The only exception seems to be the `bash.exe` for WSL, but, in that case `uname -s` returns "Linux".

@MovieStoreGuy you probably had some configuration/shell that I am missing.

Reply:
For the record: on the current GH Windows runner `uname -s` returns `MINGW64_NT-10.0-20348`.

Reply:
Thx for the details!